installer/terraform/scripts/config.py

```diff
@@ -89,6 +89,7 @@
             'SLACK': config['SLACK'],
             'SPLUNK': config['SPLUNK'],
             'APIGEE': config['APIGEE'],
-            'ACL': config['ACL']
+            'ACL': config['ACL'],
+            'AZURE': {}
```

Comment:
what is the purpose of an empty map?

Reply:
By default it will be empty. Later after applying the extension, it will update with proper values using CONFIG API.

Reply:
just seemed inconsistent from the others. will it break if we just use config['AZURE'] similar to the other values?

Reply:
config is the object read form installer configuration json file. If we need to use config['AZURE'] here, then we need to add an 'AZURE': {} code in the installer configuration file. https://github.com/tmobile/jazz-installer/blob/a3482fd0c75a3fba19a2816dce90347c1f0024bc/installer/terraform/provisioners/cookbooks/jenkins/files/default/jazz-installer-vars.json

Reply:
@devsatishm Thoughts? I think we should keep this consistent.

Reply:
Agreed. To be consistent, can we add something like below?
```
"AZURE": {
        "IS_ENABLED": "{ENABLE_AZURE}" 
}
```
Took ref from https://github.com/tmobile/jazz-installer/blob/azure-support/installer/terraform/provisioners/cookbooks/jenkins/files/default/jazz-installer-vars.json

Reply:
Yes. I will update

Reply:
@raghits @devsatishm Created one PR to handle this case - https://github.com/tmobile/jazz-installer/pull/565